Makefile

```diff
@@ -5,8 +5,9 @@ ENABLE_FEATURES ?=
 ifeq ($(TCMALLOC),1)
 ENABLE_FEATURES += tcmalloc
 else ifeq ($(SYSTEM_ALLOC),1)
-# no feature needed for system allocator
+ENABLE_FEATURES += system-alloc
 else
+# default for jemallocator
 ENABLE_FEATURES += jemalloc
```

Comment:
I don't think we need this because it is on by default.

Reply:
It is needed because every cargo invocation in the makefile passes `--no-default-features`. it does this so that the makefile doesn't have to deal with the logic of figuring out when it would otherwise need to turn off default features.